dev/include/hw/multiboot.h

```diff
@@ -172,5 +223,82 @@ enum {
     MULTIBOOT_FRAMEBUFFER_TYPE_EGA_TEXT = 2,
 };
 
+#if MULTIBOOT2_SUPPORT
+
+struct multiboot2_info {
+    uint32_t total_size;
+    uint32_t reserved;
+};
+
+struct multiboot2_tag {
+    uint32_t type;
```

Comment:
The spec seems to indicate this is actually two 16bit type and flags fields. Since it's little endian, it just so happens that type is in the bottom 16 bits and the current code doesn't deal with any flags.

Either way may make sense to follow the spec and have two fields here.